odoo/addons/base/models/ir_cron.py

```diff
@@ -102,7 +102,8 @@ def _process_jobs(cls, db_name):
                     if not job:
                         break
                     _logger.debug("job %s acquired", job['id'])
-                    cls._process_job(db, cron_cr, job)
+                    cron = odoo.registry(db_name)[cls._name]
+                    cron._process_job(db, cron_cr, job)
```

Comment:
I suggest to capitalize the variable name to not confuse it with a recordset...
And add a comment, since the first thing I would think when reading this is *why*?
```suggestion
                    # enable overridings of _process_job() on that database
                    Cron = odoo.registry(db_name)[cls._name]
                    Cron._process_job(db, cron_cr, job)
```